app/styles/modules/_button-row.scss

```diff
@@ -58,18 +58,6 @@ button {
     background: $error-background-color;
     border: 0;
     color: $message-text-color;
-
```

Comment:
This change will remove the red colors on the focus ring on the warning-button, [which is bug #6950](https://github.com/mozilla/fxa-content-server/issues/6950) rather than #6957. 

I'm not yet sure what the proper fix is for this is, I'll need to have a think and can respond on Monday if you haven't figured it out already.

Reply:
I guess we will have to explicitly write the display as block/inline-block as the <a> tag won't reduce its width otherwise. On inspect element, I found the 'forgot password' styles were in the main.css file,hence changed it